src/styles/AdminStyles/index.less

```diff
@@ -187,7 +187,6 @@ thead.ant-table-thead > tr > th {
   width:100%;
 }
 .ant-card{
```

Comment:
Be careful with changing the default antd styles directly. This can have unintended consequences in other components. The best way to avoid it is to specify which component you are changing (ex. .courseDetails .ant-card {})